webnn/idlharness.https.any.js

```diff
@@ -28,7 +28,7 @@ idl_test(
     });
 
     const operandType = {type: 'float32', dimensions: [1, 1, 5, 5]};
-    self.context = navigator.ml.createContext();
+    self.context = await navigator.ml.createContext();
```

Comment:
@foolip, `createContext()` is `sync` in [WebNN](https://webmachinelearning.github.io/webnn/#dom-ml-createcontext).